src/chainstate/stacks/events.rs

```diff
@@ -157,9 +166,9 @@ impl NFTTransferEventData {
             formatted_bytes
         };
         json!({
-            "asset_identifier": self.asset_identifier,
-            "sender": self.sender,
-            "recipient": self.recipient,
+            "asset_identifier": self.asset_identifier.json_serialize(),
```

Comment:
We should use the asset identifier representation described in the SIP - you can just use `format!("{}", self.asset_identifier)`, and get rid of the `impl AssetIdentifier`